packages/apollo-server/src/index.ts

```diff
@@ -121,11 +121,20 @@ export class ApolloServer extends ApolloServerBase {
   }
 
   public async stop() {
+    let warnOutstandingConnections = setInterval(() => {
```

Comment:
I spent too long trying to figure out why the server wasn't updating when I expected, turns out this function was blocking until I quit the Playground. Is there any way to update the schema without restarting the server? Seems like that's what normally happens, but I don't know how.